scripts/slack/notify_push.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+import requests
+import json
+
+def notifyslack( branch_name ):
+    circle_build_url = os.getenv('CIRCLE_BUILD_URL')
+    url = os.getenv('SLACK_WEBHOOK')
+
+    data = {
+      'attachments': 
+      [
+        {
+          'color': '#7CD197',
+          'fallback': 'Build Notification: ' + circle_build_url,
+          'title': 'Kubernetes-Monitor Publish Notification',
+          'text': ':egg_fancy: Published Kubernetes-Monitor: `' + branch_name + '` :egg_fancy:'
+        }
+      ]
+    }
+
+    requests.post(url, data = json.dumps(data))
+    return
```

Comment:
not sure we need the ```return``` here

Reply:
For some reason I was under the impression that every method in python needs a return statement! I just double checked and it does not, if it's skipped 'None' is returned 👍

Reply:
yes, just like ```undefined``` for Node :)